sensor_msgs/include/sensor_msgs/fill_image.hpp

```diff
@@ -32,6 +32,9 @@
 *  POSSIBILITY OF SUCH DAMAGE.
 *********************************************************************/
 
+// This file is originally ported from ROS1:
+// https://github.com/ros/common_msgs/blob/jade-devel/sensor_msgs/include/sensor_msgs/fill_image.h
```

Comment:
Nit (as this file didnt change for years): Can you please use a link to the specific commit it was copied from rather than just the branch ?

https://github.com/ros/common_msgs/blob/89069bc/sensor_msgs/include/sensor_msgs/fill_image.h

Reply:
Done: 29a141d